server/src/main/java/com/linecorp/centraldogma/server/internal/api/ContentServiceV1.java

```diff
@@ -406,15 +407,26 @@ private static Object handleWatchFailure(Throwable thrown) {
     }
 
     /**
-     * Checks if the commit is for mirroring setting and raises an exception if the {@code localRepo} field
-     * is one of {@code meta} and {@code dogma} which are internal repositories.
+     * Checks if the commit is for creating a file and raises a {@link InvalidPushException} if the
+     * given {@code repoName} field is one of {@code meta} and {@code dogma} which are internal repositories.
      */
-    public static void checkMirrorLocalRepo(String repoName, Iterable<Change<?>> changes) {
-        // TODO(minwoox): Provide an internal API for mirroring setup with a better UI(?) and check this there.
+    public static void checkPush(String repoName, Iterable<Change<?>> changes) {
+        if (Project.REPO_DOGMA.equals(repoName)) {
+            throw new InvalidPushException(
+                    "The " + Project.REPO_DOGMA + " repository is reserved for internal usage.");
+        }
```

Comment:
According to the `RequiresPermissionDecorator`, requests sent by non-admin users never reach here.
Could we remove this?

Reply:
That's correct. We can remove 414~ 417 lines. 😉

Reply:
Thanks 👍